samples/highcharts/yaxis/stackshadow/demo.js

```diff
@@ -0,0 +1,75 @@
+Highcharts.chart('container', {
+    chart: {
+        type: 'pictorial'
+    },
+
+    title: {
+        text: 'Composition of the human body'
+    },
+
+    xAxis: {
+        categories: ['Woman', 'Man'],
+        lineWidth: 0,
+        opposite: true
+    },
+
+    yAxis: {
+        visible: false,
+        stackShadow: {
+            enabled: true
+        },
+        max: 100
+    },
+
+    legend: {
+        itemMarginTop: 15,
+        itemMarginBottom: 15,
+        layout: 'vertical',
+        padding: 0,
+        verticalAlign: 'middle',
+        align: 'center',
```

Comment:
When viewing on the mobile (chart width ~400px), the legend overlaps with the columns. We need either some responsive settings or some tuning of `series pointPadding`, `groupPadding` or similar.